src/main/io/gps.h

```diff
@@ -90,6 +90,7 @@ typedef struct gpsConfig_s {
     gpsAutoBaud_e autoBaud;
     uint8_t gps_ublox_use_galileo;
     ubloxMode_e gps_ublox_mode;
+    uint8_t gps_ublox_refresh_rate_hz;
```

Comment:
You cannot alter the existing structure of the parameter group. The new value must be added to the end. See https://github.com/betaflight/betaflight/blob/master/docs/development/ParameterGroups.md

Reply:
Moved to the end.